lock-api-objects/src/main/java/com/palantir/lock/LockRequest.java

```diff
@@ -428,8 +428,8 @@ public String toString() {
                     + ", blockingDuration=" + blockingDuration //
                     + ", versionId=" + versionId //
                     + ", lockGroupBehavior=" + lockGroupBehavior //
-                    + ", lockCount=" + lockMap.size() //
-                    + ", firstLock=" + Iterables.getFirst(lockMap.entries(), "") //
+                    + ", lockCount=" + (lockMap == null ? 0 : lockMap.size()) //
+                    + ", firstLock=" + (lockMap == null ? null : Iterables.getFirst(lockMap.entries(), ""))
```

Comment:
this makes sense if we were previously seeing NPEs here, though curious if there was a separate issue that prompted this?

Reply:
this was just proactive prevention to avoid possible NPE as these fields are technically nullable